specification/configuration.md

```diff
@@ -135,7 +135,11 @@ beyond the OpenTelemetry specification exist.
 - `OTEL_TRACES_EXPORTER`
   - Non-RUM distribution MUST default to `otlp` over gRPC with an endpoint of `localhost:4317`
   - Non-RUM distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
-
+- `OTEL_LOG_LEVEL`
+  - Distribution SHOULD support `DEBUG` setting. With this value all relevant troubleshooting data should be logged. 
```

Comment:
AFAIK supporting this would involve hacking in a most of SDKs

Reply:
https://github.com/open-telemetry/opentelemetry-specification/issues/920

Reply:
Yeah, we need to wait for the values to be finalized in the spec. Then it will be a no-brainer ;)